src/utilities/cities/index.ts

```diff
@@ -0,0 +1,24 @@
+import { CITIES_DATA } from '../../common/cities';
+
+import { State, getStates } from '../states';
+
+const sortAlphabetically = (cityA: string, cityB: string) => cityA.localeCompare(cityB);
+
+export function getCities({ state: stateFilter }: { state?: Partial<State> } = {}): string[] {
+  if (stateFilter) {
+    const states = getStates();
```

Comment:
What do you think to dynamic require `getStates` just if receive a filter?

Reply:
I don't know. I've never used this way. What would be the advantage of a dynamic require?

Reply:
With dynamic require on the browser, we just import `getStates` when filter is present. I think that is not important right now and we could change in the future.